### DISCUSSION TITLE
A globalconfig way to use global scope

### DISCUSSION THREAD

**Discussion by volarname** [intlify/vue-i18n#936] [Q&A]
hello, its possible to define to use global scope globally?
i dont want to use `const { t } = useI18n({ useScope: 'global' })` in each component
the option to disable warning will be also great - and put in the warning at least translation key, so its easier to identify place

**Comment by kazupon** [comment]
you can use `globalInjection: true`
See the details: https://vue-i18n.intlify.dev/guide/advanced/composition.html#implicit-with-injected-properties-and-functions

**Comment by volarname** [comment]
@kazupon but i dont want global injection and to use $t in my components
your reply is unrelated
i am using script setup and composition api and always i need to specify `const { t } = useI18n({ useScope: 'global' })`
a way how to do it only using `const { t } = useI18n()` + some global config should exist

**Comment by volarname** [comment]
or at least add possibility to disable scope warnings in console
